### DISCUSSION TITLE
Mac with M1

### DISCUSSION THREAD

**Discussion by saulrazom** [mortbopet/Ripes#295] [Q&A]
¿Which is the best way to run Ripes when having a MacOS with M1 or M2? (Emulator doesn´t suffice my needs)

**Comment by mortbopet** [comment]
CC @atw1020 @mheese who may have some input here.

**Reply by mheese** [reply]
Unfortunately, I guess at the moment the only way is to compile it from source and run the executable directly. I've followed @atw1020 's problem on the packaged universal app (https://github.com/mortbopet/Ripes/pull/276), but I'm also drawing a blank here on how to fix that.
